docs/en/05-supported-backends/vacc.md

```diff
@@ -0,0 +1,92 @@
+# VACC Backend
+
+- cmake 3.10.0+
+- gcc/g++ 7.5.0
+- llvm 9.0.1
+- ubuntu 18.04
+
+## PCIE
+
+### 1.package
+
+- dkms (>=1.95)
+- linux-headers
+- dpkg (Ubuntu)
+- rpm  (CentOS)
+- python2
+- python3
+
+Check if there is a vacc card：`lspci -d:0100`
+
+1. Requirements
+
+   ```bash
+   sudo apt-get install dkms dpkg python2 python3
+   ```
+
+2. install driver
+
+   ```bash
+   sudo dpkg -i vastai-pci_xx.xx.xx.xx_xx.deb
+   ```
+
+3. Verify installation
+
+   ```bash
+   dpkg --status vastai-pci-xxx
+
+   #output
+   Package: vastai-pci-dkms
+   Status: install ok installed
+   ……
+   Version: xx.xx.xx.xx
+   Provides: vastai-pci-modules (= xx.xx.xx.xx)
+   Depends: dkms (>= 1.95)
+   Description: vastai-pci driver in DKMS format.
+
+   lsmod | grep vastai_pci
+
+   #output
+   vastai_pci        xxx  x
+   ```
+
+4. Upgrade driver
+
+   ```bash
+   sudo dpkg -i vastai-pci_dkms_xx.xx.xx.xx_xx.deb
+   ```
+
+5. Uninstall driver
+
+   ```bash
+   sudo dpkg -r vastai-pci_dkms_xx.xx.xx.xx_xx
+   ```
+
+### 2.reboot pcie
+
+```bash
+sudo chmod 666 /dev/kchar:0 && sudo echo reboot > /dev/kchar:0
+```
+
+## SDK
+
+### step.1
+
+```bash
+pip install -r requirements.txt
+```
+
+### step.2
+
+```bash
+sudo vi ~/.bashrc
+
+export VASTSTREAM_PIPELINE=true
+export VACC_IRTEXT_ENABLE=1
+export TVM_HOME="/opt/vastai/vaststream/tvm"
```

Comment:
Is TVM installed automatically?

Reply:
No. we will provide vacc backend environment package, which contain tvm package. this tvm is not that opensource tvm.